--- origsrc/hdate-applet-0.15.10/Makefile.am	2008-06-29 22:27:41.000000000 -0500
+++ src/hdate-applet-0.15.10/Makefile.am	2008-07-01 01:17:54.281250000 -0500
@@ -37,4 +37,4 @@
 Applications_DATA = ghcal.desktop
 
 hdate-applet.server:
-	sed -e "s|\@LIBEXECDIR\@|$(libexecdir)|" hdate-applet.server.in.in > hdate-applet.server
+	sed -e "s|\@LIBEXECDIR\@|$(libexecdir)|" $(srcdir)/hdate-applet.server.in.in > hdate-applet.server
